app-emulation/img/img-0.5.11.ebuild

```diff
@@ -0,0 +1,34 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+EGO_PN="github.com/genuinetools/img"
+
+inherit golang-vcs-snapshot
```

Comment:
Looking at upstream, you should be using the *go-module.eclass* with this package.
https://devmanual.gentoo.org/eclass-reference/go-module.eclass/index.html

Reply:
I will try to fix this as soon as possible. Thank you for the review :)